datalake/src/main/resources/duties/7.2.7/aws/medium_duty_ha.json

```diff
@@ -44,7 +84,7 @@
         }
       },
       "nodeCount": 1,
-      "type": "GATEWAY",
+      "type": "CORE",
```

Comment:
We have "core" of "CORE" type and "main"/"auxiliary" of "CORE" type. Very confusing :)

Reply:
Well, we started with gateway as gateway.  Might as well continue the confusion.